### PR TITLE
Unpack war file in docker image to accelerate startup.

### PR DESCRIPTION
**Pull request by lablans**
**What's in the PR**
* Rather than unpacking .war at each start, unpack once during docker build.

**How to test manually**
* Build as usual, see that Connector is executed. Actual code is unchanged.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -6,7 +6,11 @@ ARG PROJECT=samply
 ENV PROJECT=$PROJECT
 
 RUN ["rm", "-fr", "/usr/local/tomcat/webapps"]
-ADD target/connector.war                        /usr/local/tomcat/webapps/ROOT.war
+
+RUN set -x ; \
```
Comment (alexanderkiel) [inline Dockerfile:10]: Can we please move the introduction of a separate user into its own PR? At the end, tomcat is still executed as root. If we move to a non-root user, we should do it everywhere.

**Review by alexanderkiel — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (lablans) [reply Dockerfile]: Right -- the user was introduced in the Mainzelliste docker container to support Tomcat running as non-root. Seems like the official Tomcat image runs as root anyways. I've just checked: Tomcat extracts everything as root:root, so I've now mimicked that behaviour. PR updated.

**Review by alexanderkiel — APPROVED** [review]
